django/db/models/sql/query.py

```diff
@@ -104,6 +106,8 @@ def get_child_with_renamed_prefix(prefix, replacement, child):
         child = child.copy()
         if child.name.startswith(prefix + LOOKUP_SEP):
             child.name = child.name.replace(prefix, replacement, 1)
+    elif isinstance(child, QuerySet):
+        pass
```

Comment:
I'd suggest using duck typing with `.subquery` instead.

```suggestion
    elif getattr(child, "subquery", False):
        pass
```

Reply:
Ah it's possibly not doable because we're dealing with unresolved expression at this point.